source/Reloaded.Mod.Launcher/Assets/Languages/en-GB.xaml

```diff
@@ -36,6 +36,7 @@
     <sys:String x:Key="AddAppName">Application Name</sys:String>
     <sys:String x:Key="AddAppLocation">Executable Location</sys:String>
     <sys:String x:Key="AddAppArguments">Commandline Arguments</sys:String>
+    <sys:String x:Key="AddAppWorkingDirectory">Working Directory</sys:String>
```

Comment:
Please move this one to the bottom of the file, and add next version number to it (1.23.8);
at one point I moved to using chronological order to make localisation easier.